dev/io.openliberty.mcp.internal_fat/build.gradle

```
@@ -15,12 +15,38 @@ apply from: '../wlp-gradle/subprojects/repos.gradle'
 configurations.requiredLibs {
   transitive = false
 }
+
+sourceSets {
+  // Source set where we don't include parameter names
+  noParamNamesTest {
+    java {
+      srcDirs = ['noParamTest']
+    }
+  }
+}
```

```suggestion
sourceSets {
  // Source set where we don't include parameter names
  noParamNames {
    java {
      srcDirs = ['noParamNames']
    }
  }
}
```

Let's make the names consistent - other parts of this file will need to change to match.

We might want a README file within `noParamNames` explaining what it's for.

---

Done, and added ReadMe